packages/react-native-macos-init/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "react-native-macos-init",
+    "version": "0.0.9",
+    "description": "CLI to add react-native-macos to an existing react-native project",
+    "main": "index.js",
+    "repository": "https://github.com/microsoft/react-native-macos",
+    "license": "MIT",
+    "private": false,
+    "scripts": {
+      "build": "just-scripts build",
+      "clean": "just-scripts clean",
+      "lint": "just-scripts lint",
+      "lint:fix": "just-scripts lint:fix"
```

Comment:
Maybe add the following so it tries to automatically build before publishing the package–which I did and ended up with an empty package in my local registry proxy.

```json
"prepublishOnly": "npm run build"
```